test/integration/testem/testem.js

```diff
@@ -118,7 +118,7 @@ const ciTestemConfig = {
     "tap_quiet_logs": true,
     "browser_args": {
         "Chrome": {
-            "ci": [ "--disable-backgrounding-occluded-windows", "--ignore-gpu-blocklist", "--use-gl=desktop" ]
+            "ci": [ "--disable-backgrounding-occluded-windows", "--ignore-gpu-blocklist", "--use-angle=gl"]
```

Comment:
Hi @karimnaaji! In order to allow covering different ANGLE backends, an option to allow setting the --use-angle chrome flag locally when running render tests locally has been enabled. By default, running the render tests locally will not set any chrome flags. I did not add any CI builds to run different configurations, but it could be something added in the future if it feels helpful.

I don't think Macs default to metal for the render tests for the most part, so my assumption is possibly incorrect on metal causing render test differences. I ran `chrome://gpu/` on my computer (intel Mac) and Metal is shown as "disabled". I also had @stepankuzmin (with M1 mac) check `chrome://gpu/` and metal appears as disabled as well. However, @stepankuzmin seemed to have less failures with forcing ANGLE to use OpenGL with using the chrome flag `--use-angle=gl`, so I'm not entirely sure of the cause there but allowing to pass the use-angle flag options locally could be helpful and to also test out different graphics backends as well.